composer.json

```diff
@@ -11,7 +11,8 @@
     },
     "minimum-stability": "dev",
     "require": {
-        "phergie/phergie-irc-bot-react": "~1"
+        "phergie/phergie-irc-bot-react": "~1",
+        "phergie/phergie-irc-parser": "~1.6"
```

Comment:
I'd rather just update the version number on the bot, but if there is not a tagged version of the bot that meets our needs, this is an acceptable solution.